app/views/courses/show.html.haml

```diff
@@ -19,6 +20,7 @@
             %td= @course.start_date
             %td= @course.end_date
             %td= @course.allocation
+            %td= @course.institute
```

Comment:
What does @course.institute output? I can't find a to_s. 
If it prints the institute name, why not use that above in the index page?

Reply:
Ah, thanks. That should have been `@course.institute.name` - totally missed that